libraries/SRV_Channel/SRV_Channel.cpp

```diff
@@ -122,13 +131,18 @@ void SRV_Channel::calc_pwm(int16_t output_scaled)
     } else {
         pwm = pwm_from_range(output_scaled);
     }
+    // If not in unlimited ouput we contrain the ouput
+    if (unlimited_pwm_mask & (1U << ch_num)) {
```

Comment:
Ah, sorry it's a mask of which channels to not constrain... That's a confusing variable name then. It should be called limited_pwm_mask or something, unlimited to me implies that if the bit is set the channel isn't limited, which is why I got it wrong.